pdr_backend/trader/trader_config.py

```diff
@@ -10,4 +10,5 @@ class TraderConfig(BaseConfig):
     def __init__(self):
         super().__init__()
 
-        self.trader_min_buffer = int(getenv("TRADER_MIN_BUFFER", "60 "))
+        self.trader_min_buffer = int(getenv("TRADER_MIN_BUFFER", "60"))
+        self.max_tries = int(getenv("TRADER_MAX_TRIES", "10"))
```

Comment:
I don't understand.

Why are there two config variables being set by one envvar?

And what is "trader min buffer" anyway?

As mentioned : we don't need to have an envvar for every parameter. We should hardcode some vars in the config for now: ones that don't need much changing because they have a very good default.

Reply:
It's not always better to make an envvar and make a default. Every envvar adds complexity because you have to document it elsewhere.

Hardcoding is fine, a good thing actually, in specially fenced locations. These are: constants.py, config classes, and "ss" (solution strategy) classes. You'll see as classes coming in when Calina updates predictoor approach 2.

Reply:
Btw: you need to document in the code what trader min buffer means. Just a short one liner above where it's set.